lrlex/src/lib/ctbuilder.rs

```diff
@@ -403,7 +403,7 @@ where
         write!(
             outs,
             "{mod_vis} mod {mod_name} {{
-use lrlex::{{LexerDef, LRNonStreamingLexerDef, Rule, StartState}};
+use lrlex::{{LexerDef, LRNonStreamingLexerDef, Rule, StartState, StartStateOperation}};
```

Comment:
I think we were moving to fully qualified paths here in generate code, so i believe this should be `::lrlex::StartStateOperation` below, besides these minor things I think this is looking good!

Reply:
Fixed under ddaae44153ae8441522a4b20b8a7593f22a9eb6c